nvme.c

```diff
@@ -2822,26 +2822,42 @@ static int id_endurance_grp_list(int argc, char **argv, struct command *cmd,
 	return err;
 }
 
-static void ns_mgmt_show_error(struct nvme_dev *dev, int err, const char *cmd)
+static bool is_ns_mgmt_support(struct nvme_dev *dev)
 {
-	_cleanup_free_ struct nvme_id_ctrl *ctrl = NULL;
+	int err;
 
+	_cleanup_free_ struct nvme_id_ctrl *ctrl = nvme_alloc(sizeof(*ctrl));
+
+	if (ctrl) {
+		err = nvme_cli_identify_ctrl(dev, ctrl);
+		if (err)
+			goto out;
+		if (le16_to_cpu(ctrl->oacs) & NVME_CTRL_OACS_NS_MGMT)
+			return true;
+		return false;
+	}
+out:
+	return true;
+}
+
+static void ns_mgmt_show_status(struct nvme_dev *dev, int err, const char *cmd, char *name,
+				__u32 nsid)
+{
 	if (err < 0) {
 		nvme_show_error("%s namespace: %s", cmd, nvme_strerror(errno));
 		return;
 	}
 
 	nvme_show_init();
 
-	nvme_show_status(err);
-
-	ctrl = nvme_alloc(sizeof(*ctrl));
-	if (!ctrl)
-		return;
-
-	err = nvme_cli_identify_ctrl(dev, ctrl);
 	if (!err) {
-		if (!(le16_to_cpu(ctrl->oacs) & NVME_CTRL_OACS_NS_MGMT))
+		nvme_show_key_value(true, false, name, "Success, %s",
+				    !strcmp(cmd, "create") ? "created" :
+				    !strcmp(cmd, "delete") ? "deleted" : "");
```

Comment:
strcmp `cmd` seems a bit excessive, why not just write 'create command was sucessful' or something along the line. or just 'success'. `nvme create-ns` -> `success` should be good enough.